### PR TITLE
Add remaster spells previewed in the Paizo blog

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: packs/spells/tree-of-seasons.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            "somatic": true
+        },
+        "cost": {
+            "value": ""
+        },
+        "damage": {
+            "value": {}
+        },
+        "description": {
+            "value": "<p>You cause a Small tree to instantly sprout in an unoccupied space on the ground. Four seedpods grow from the tree, each filled with the magic of a different one of the four seasons. A creature can Interact to pluck one of the pods, and can then either throw it up to 30 feet as part of the same action or do so with a separate Interact action later. When thrown, a pod explodes in a @Template[type:burst|distance:5], dealing 6d6 damage with a basic Reflex save against your spell DC. The damage type depends on the season of the pod: electricity for spring, fire for summer, poison for autumn, or cold for winter. When the spell ends, the tree withers away and any remaining pods rot, leaving behind non-magical seeds.</p>\n<p>@Damage[(@item.level)d6[cold]]</p>\n<p>@Damage[(@item.level)d6[electricity]]</p>\n<p>@Damage[(@item.level)d6[fire]]</p>\n<p>@Damage[(@item.level)d6[poison]]</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The burst's damage increases by 1d6.</p>"
```
Comment (simonward) [inline packs/spells/tree-of-seasons.json:20]: This might be better handled with overlays?

File: packs/spells/read-aura.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
             "value": {}
         },
         "description": {
-            "value": "<p>You focus on the target object, opening your mind to perceive magical auras. When the casting is complete, you know whether that item is magical, and if it is, you learn the school of magic.</p>\n<p>If the object is illusory, you detect this only if the effect's level is lower than the level of your read aura spell.</p>\n<hr />\n<p><strong>Heightened (3rd)</strong> You can target up to 10 objects.</p>\n<p><strong>Heightened (6th)</strong> You can target any number of objects.</p>"
+            "value": "<p>You focus on the target object, opening your mind to perceive magical auras. When the casting is complete, you know whether that item is magical. You or anyone you advise about the aura gains a +2 circumstance bonus to @UUID[Compendium.pf2e.actionspf2e.eReSHVEPCsdkSL4G]{Identify Magic} on the item. If the object is illusory, you detect this only if the effect's rank is lower than the rank of your <em>read aura</em> spell.</p>\n<hr />\n<p><strong>Heightened (3rd)</strong> You can target up to 10 objects.</p>\n<p><strong>Heightened (6th)</strong> You can target any number of objects.</p>"
```
Comment (simonward) [inline packs/spells/read-aura.json:24]: Old style link.

File: packs/spells/tree-of-seasons.json (hunk 1 above)
Comment (TikaelSol) [reply packs/spells/tree-of-seasons.json:20]: Each casting is all four damage types, one pod of each.

File: packs/spells/read-aura.json (hunk 2 above)
Comment (TikaelSol) [inline packs/spells/read-aura.json:24]:
```suggestion
            "value": "<p>You focus on the target object, opening your mind to perceive magical auras. When the casting is complete, you know whether that item is magical. You or anyone you advise about the aura gains a +2 circumstance bonus to @UUID[Compendium.pf2e.actionspf2e.Item.eReSHVEPCsdkSL4G]{Identify Magic} on the item. If the object is illusory, you detect this only if the effect's rank is lower than the rank of your <em>read aura</em> spell.</p>\n<hr />\n<p><strong>Heightened (3rd)</strong> You can target up to 10 objects.</p>\n<p><strong>Heightened (6th)</strong> You can target any number of objects.</p>"
```

File: packs/spells/tree-of-seasons.json (hunk 1 above)
Comment (simonward) [reply packs/spells/tree-of-seasons.json:20]: Right you are.

File: packs/spells/read-aura.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
             "value": {}
         },
         "description": {
-            "value": "<p>You focus on the target object, opening your mind to perceive magical auras. When the casting is complete, you know whether that item is magical, and if it is, you learn the school of magic.</p>\n<p>If the object is illusory, you detect this only if the effect's level is lower than the level of your read aura spell.</p>\n<hr />\n<p><strong>Heightened (3rd)</strong> You can target up to 10 objects.</p>\n<p><strong>Heightened (6th)</strong> You can target any number of objects.</p>"
+            "value": "<p>You focus on the target object, opening your mind to perceive magical auras. When the casting is complete, you know whether that item is magical. You or anyone you advise about the aura gains a +2 circumstance bonus to @UUID[Compendium.pf2e.actionspf2e.Item.eReSHVEPCsdkSL4G]{Identify Magic} on the item. If the object is illusory, you detect this only if the effect's rank is lower than the rank of your <em>read aura</em> spell.</p>\n<hr />\n<p><strong>Heightened (3rd)</strong> You can target up to 10 objects.</p>\n<p><strong>Heightened (6th)</strong> You can target any number of objects.</p>"
```
Comment (TikaelSol) [inline packs/spells/read-aura.json:24]:
```suggestion
            "value": "<p>You focus on the target object, opening your mind to perceive magical auras. When the casting is complete, you know whether that item is magical. You or anyone you advise about the aura gains a +2 circumstance bonus to @UUID[Compendium.pf2e.actionspf2e.Item.Identify Magic] on the item. If the object is illusory, you detect this only if the effect's rank is lower than the rank of your <em>read aura</em> spell.</p>\n<hr />\n<p><strong>Heightened (3rd)</strong> You can target up to 10 objects.</p>\n<p><strong>Heightened (6th)</strong> You can target any number of objects.</p>"
```

**Review by simonward — APPROVED** [review]

**Pull request merged**